### ISSUE TITLE
proposed updated procdesc and supplinf verbiage

### ISSUE THREAD

**Issue by forkozi** [noaa-rsd/LiProd#28]
https://github.com/noaa-rsd/LiProd/blob/0f4209e08ca5637f6243966554b12097d7d5fbb1/support_files/library.json#L21-L22

Hey y'all (@gimahori  and @stephen-a-white ),

I've got some potential new verbiage I wanted to share.   I tried to capture more fully the new PDAL/VDatum workflow in the \<procdesc\> and \<supplinf\> fields for product 2:

>"procdesc": "The lidar point data were processed to an initial LAS format using the Riegl RiProcess. Applanix POSPac was used to process the trajectory, and GeoCue, Terrasolid, and LP360 were used for data visualization, 3D editing, and export to final LAS \<LAS VERSION\> files. PDAL was used to create an initial 1-m resolution, ellipsoidally referenced DEM from the following classes: ground (\<GROUND CLASS\>), bathymetric point (\<BATHY CLASS\>), submerged object (\<SUBMERGED CLASS\>), and International Hydrographic Organization (IHO) S-57 object (\<IHO CLASS\>). PDAL was configured to use mean resampling, with a sampling radius of square_root(2).  An automated ground classification algorithm was used to determine bare-earth point classification. The ellipsoidaly referenced DEM was then vertically transformed to <CHART_DATUM> heights using a co-registered datum-separation model generated with VDatum. The values in the DEM were not rounded (no rounding rule was applied)."
		
>"supplinf": "This Geotiff has three bands: 1) Chart-Datum Heights (\<CHART_DATUM\>), 2) Total Horizontal Uncertainty (THU), and 3) Total Vertical Uncertainty (TVU).  The THU band was generated by first using cBLUE (https://noaa-rsd.github.io/cBLUE.github.io/) to calculate per-point propagated horizontal uncertainties and then using PDAL to grid the cBLUE THU values at a 1-m resolution using mean resampling with a sampling radius of square_root(2).  The TVU band was generated by combining in quadrature a TVU grid (generated using the same method used to generate the THU band) and the co-registered maximum cumulative uncertainty (MCU) model of the VDatum vertical-transformation model characterizing the separation between the ellipsoid and the specified chart datum.  Please refer to https://vdatum.noaa.gov/docs/est_uncertainties.html for documentation on MCU."

What do ya think?  Too wordy...not enough...ok?

By the way...are there certain types of things that go in <procdesc\> versus \<supplinf\>?  I somewhat arbitrarily split up information.

Pending y'alls feedback, I can update the corresponding fields for Product 1 and 3, too.

**Comment by stephen-a-white** [comment]
I think it looks good.  I think the <procdesc> is the entry for the how and what in regards to processing the data.  The <supplinf> is just additional information you might want the user to know.

Looks Good!

**Comment by forkozi** [comment]
Hey, I wanted to reopen this one to get your opinion on something.  What do you think about rearranging the content of the \<procdesc\> and \<supplinf\> fields?  Right now, there is information about how the tiffs were made in both fields.  Could we consolidate the tiff process into the \<procdesc\> field (because the tiff is the actual product the xml is documenting) and then put the information about the underlying LAS data in the  \<supplinf\> field?

**Comment by forkozi** [comment]
\<procdesc\> 
>"This 1m-resolution Geotiff has three bands: 1) Chart-Datum Heights (<CHART_DATUM>), 2) Total Horizontal Uncertainty (THU), and 3) Total Vertical Uncertainty (TVU).  Band 1 was generated in two steps.  First, PDAL was used to create an initial, ellipsoidally referenced DEM from the following classes: ground (<GROUND CLASS>), bathymetric point (<BATHY CLASS>), and submerged object (<SUBMERGED CLASS>), using mean resampling with a sampling radius of square_root(2). Second, the ellipsoidaly referenced DEM was then vertically transformed to <CHART_DATUM> heights using a co-registered datum-separtion model generated with VDatum. Band 2 was generated in two steps.  First, per-point propagated horizontal uncertainties were calcuated using cBLUE (https://noaa-rsd.github.io/cBLUE.github.io/). Second, PDAL was used to grid the cBLUE THU values at a 1-m resolution using mean resampling with a sampling radius of square_root(2). Band 3 was generated in three steps. Similar to Band 2, cBLUE was first used to calcuate per-point propagated vertical uncertainties, and then PDAL was used to grid the cBLUE TVU values at a 1-m resolution using mean resampling with a sampling radius of square_root(2). Third, the PDAL-gridded TVU layer was combined in quadrature with a co-registered maximum cumulative uncertainty (MCU) model of the VDatum vertical-transformation model characterizing the separation between the ellipsoid and the specified chart datum.  Please refer to https://vdatum.noaa.gov/docs/est_uncertainties.html for documentation on MCU."

\<supplinf\>
>"The lidar point data used to generate this product were processed to an initial LAS format using the Riegl RiProcess software. Applanix POSPac was used to process the trajectory, and GeoCue, Terrasolid, and LP360 were used for data visualization, 3D editing, and export to final LAS <LAS VERSION> files. An automated ground classification algorithm was used to determine bare-earth point classification."

**Comment by stephen-a-white** [comment]
Or we could just place it all in <procdesc> and leave the <supplinf> blank or remove.  I also do not have a problem leaving it the way you proposed either.  Seems to make sense.

**Comment by forkozi** [comment]
do you have a preference?   I could go ahead and pick one, but I'd be able
to sleep better at night if you make the call!  I think you've got a much
better handle on the history of these fields and how they're useful
downstream.

**Comment by stephen-a-white** [comment]
Lets go with what you proposed.  I think it looks good.  Would not loose sleep over it.
